app/src/test/java/ch/epfl/sdp/items/MarketTest.java

```diff
@@ -0,0 +1,99 @@
+package ch.epfl.sdp.items;
+
+
+import androidx.core.util.Pair;
```

Comment:
Previously this was android.util.Pair however this failed as it was a deprecated class, so I changed it to this (lost 3 hours of my time until I figured out it was a problem with the implementation of android.util.Pair)